File: apiForMagazyn/src/main/java/org/apiForMagazyn/api/company/Company.java
package org.apiForMagazyn.api.company;

import lombok.Getter;
import lombok.Setter;
import org.apiForMagazyn.api.adres.Adres;

import javax.persistence.*;
import javax.validation.constraints.Size;

@Entity
@Table
@Setter
@Getter
public class Company {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer idCom;

    @Size(max = 50)
    private String name;

    @Size(max = 50)
    private String person;

    @Size(max = 20)
    private String nip;
    @OneToOne
    @JoinColumn(name="idAd")
    private Adres adres;
}

File: Magazyn/Magazyn/Tables/Company.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Magazyn.Tables
{
    class Company
    {
        public int idCom { set; get; }
        public string name { set; get; }
        public string person { set; get; }
        public string nip { set; get; }
        public Adres adres { set; get; }
    }
}
